webapp/src/action_types/index.js

```diff
@@ -32,4 +32,7 @@ export default {
 
     RECEIVED_CHANNEL_SUBSCRIPTIONS: `${PluginId}_recevied_channel_subscriptions`,
     DELETED_CHANNEL_SUBSCRIPTION: `${PluginId}_deleted_channel_subscription`,
+
+    RECEIVED_JIRA_TICKETS: `${PluginId}_received_jira_tickets`,
+    RECEIVED_JIRA_TICKETS_ERROR: `${PluginId}_received_jira_tickets_error`,
```

Comment:
I think we only fetch one issue at a time right?

```suggestion
    RECEIVED_JIRA_TICKET: `${PluginId}_received_jira_ticket`,
    RECEIVED_JIRA_TICKET_ERROR: `${PluginId}_received_jira_ticket_error`,
```